docs/docs/getting-started.md

```diff
@@ -12,52 +12,10 @@ The easiest way to start hacking on React is using the following JSFiddle Hello
  * **[React JSFiddle](https://jsfiddle.net/reactjs/69z2wepo/)**
  * [React JSFiddle without JSX](https://jsfiddle.net/reactjs/5vjqabv3/)
 
-## Using React from npm
 
-We recommend using React with a CommonJS module system like [browserify](http://browserify.org/) or [webpack](https://webpack.github.io/). Use the [`react`](https://www.npmjs.com/package/react) and [`react-dom`](https://www.npmjs.com/package/react-dom) npm packages.
+## Starter Pack
 
-```js
-// main.js
-var React = require('react');
-var ReactDOM = require('react-dom');
-
-ReactDOM.render(
-  <h1>Hello, world!</h1>,
-  document.getElementById('example')
-);
-```
-
-To install React DOM and build your bundle with browserify:
-
-```sh
-$ npm install --save react react-dom babelify babel-preset-react
-$ browserify -t [ babelify --presets [ react ] ] main.js -o bundle.js
-```
-
-To install React DOM and build your bundle with webpack:
-
-```sh
-$ npm install --save react react-dom babel-preset-react
-$ webpack
-```
-
-> Note:
->
-> If you are using ES2015, you will want to also use the `babel-preset-es2015` package.
-
-**Note:** by default, React will be in development mode, which is slower, and not advised for production. To use React in production mode, set the environment variable `NODE_ENV` to `production` (using envify or webpack's DefinePlugin). For example:
-
-```js
-new webpack.DefinePlugin({
-  "process.env": {
-    NODE_ENV: JSON.stringify("production")
-  }
-});
-```
-
-## Quick Start Without npm
-
-If you're not ready to use npm yet, you can download the starter kit which includes prebuilt copies of React and React DOM.
+If you're just getting started, you can download the starter kit which includes prebuilt copies of React and React DOM.
```

Comment:
Let's make this a bit more welcoming and say that it comes with a number of examples, maybe say those prebuilt copies of React are for the browser.